web/src/app/modules/views/main/editors/modules/contents-container/components/ceg-model-container.component.ts

```diff
@@ -31,21 +32,39 @@ export class CEGModelContainer extends TestSpecificationContentContainerBase<CEG
     }
 
     modelDescription: string;
+    objectifIndicator = 'WENN';
 
+    public isObjectif(): boolean {
+        let desc = this.modelDescription;
+
+        if (desc == undefined) {
+            desc = '';
+        }
+        desc = desc.trim();
+
+        if (desc.length <= 3) {

```

Comment:
Would it make sense to set this to 4?  This means as soon as someone types the whole word "WENN" the font is changed.
Also you could make the code much more simple by just doing description.startsWith(this.objectifIndicator)